### ISSUE TITLE
chart.js module has no exported member SankeyDataPoint

### ISSUE THREAD

**Issue by dtslvr** [kurkle/chartjs-chart-sankey#97]
We are using `chartjs-chart-sankey` version `0.12.0`. After upgrading `chart.js` from version `4.0.1` to `4.3.0`, I'm getting the following errror for `import { SankeyDataPoint } from 'chart.js';`:
```
Module '"chart.js"' has no exported member 'SankeyDataPoint'.
```

Do you have any idea why it stopped working? Thanks for any advice.

**Comment by kurkle** [comment]
I would think it has to do with that issue: https://github.com/chartjs/Chart.js/issues/11288

But maybe you should be importing it from 'chartjs-chart-sankey' instead?

**Comment by stockiNail** [comment]
> But maybe you should be importing it from 'chartjs-chart-sankey' instead?

I agree with @kurkle. The module augmentation needs anyway the import of `chartjs-chart-sankey`.

**Comment by dtslvr** [comment]
> I would think it has to do with that issue: [chartjs/Chart.js#11288](https://github.com/chartjs/Chart.js/issues/11288)

Thanks for the link, that does seem to be the problem.

> But maybe you should be importing it from 'chartjs-chart-sankey' instead?

How can I import `SankeyDataPoint`? `import { SankeyDataPoint } from 'chartjs-chart-sankey';` does not work as it is not exported.

**Comment by dtslvr** [comment]
@stockiNail I have just updated to `chart.js` `4.3.2`, but `import { SankeyDataPoint } from 'chart.js';` is still not compiling. I Would be glad to receive any assistance.

**Comment by sandranrivas** [comment]
for me updating chartjs-adapter-dayjs-4 to the latest version, solved the problem with sankeydatapoint. maybe try updating the chartjs versions you have

**Comment by sandranrivas** [comment]
any update on this issue?

**Comment by dtslvr** [comment]
Thanks, @kurkle! 👍🏻
